src/lib/product_reviews/review/header.js

```diff
@@ -34,14 +34,8 @@ const Header = ({ rating, headline, date, productName, productVariantName }) =>
 
    return (
       <React.Fragment>
-         <Headline itemProp="name">
-            <StarsContainer
-               itemProp="reviewRating"
-               itemScope
-               itemType="http://schema.org/Rating">
-               <meta itemProp="worstRating" content="1" />
-               <meta itemProp="ratingValue" content={rating} />
-               <meta itemProp="bestRating" content="5" />
```

Comment:
I see there is a `worstRating` here but we aren't including one when building `schemaReviews` now. Is that intentional?

Reply:
We can put any data we want in here that is supported, but from the [docs](https://schema.org/AggregateRating)

`If worstRating is omitted, 1 is assumed.`